pkg/controller/plan/adapter/ovirt/validator.go

```diff
@@ -105,20 +108,79 @@ func (r *Validator) StorageMapped(vmRef ref.Ref) (ok bool, err error) {
 			vmRef.String())
 		return
 	}
+
 	for _, da := range vm.DiskAttachments {
 		if da.Disk.StorageType != "lun" {
 			if !r.plan.Referenced.Map.Storage.Status.Refs.Find(ref.Ref{ID: da.Disk.StorageDomain}) {
 				return
 			}
-		} else if len(da.Disk.Lun.LogicalUnits.LogicalUnit) > 0 && da.Disk.Lun.LogicalUnits.LogicalUnit[0].Address == "" {
-			// Have LUN disk but without the relevant data. This might happen with older oVirt versions.
-			return
 		}
 	}
 	ok = true
 	return
 }
 
+// Validates oVirt version in case we use direct LUN/FC storage
+func (r *Validator) DirectStorage(vmRef ref.Ref) (ok bool, err error) {
+	if r.plan.Referenced.Map.Storage == nil {
```

Comment:
when does it happen that `r.plan.Referenced.Map.Storage == nil`?

Reply:
ok, that's even more misleading then since we don't validate the storagemap here...
so do we actually need to check it here?